docker/Dockerfile.1-13-gpu

```diff
@@ -0,0 +1,65 @@
+ARG TFS_VERSION
+
+FROM tensorflow/serving:${TFS_VERSION}-gpu as tfs
+FROM nvidia/cuda:10.0-base-ubuntu16.04
+LABEL com.amazonaws.sagemaker.capabilities.accept-bind-to-port=true
+
+COPY --from=tfs /usr/bin/tensorflow_model_server /usr/bin/tensorflow_model_server
+
+# https://github.com/tensorflow/serving/blob/1.13.0/tensorflow_serving/tools/docker/Dockerfile.gpu
+ENV CUDNN_VERSION=7.4.1.5
+ENV TF_TENSORRT_VERSION=5.0.2
+
+RUN \
+    apt-get update && apt-get install -y --no-install-recommends \
+        ca-certificates \
+        cuda-command-line-tools-10-0 \
+        cuda-command-line-tools-10-0 \
+        cuda-cublas-10-0 \
+        cuda-cufft-10-0 \
+        cuda-curand-10-0 \
+        cuda-cusolver-10-0 \
+        cuda-cusparse-10-0 \
+        libcudnn7=${CUDNN_VERSION}-1+cuda10.0 \
+        libgomp1 && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/*
+
+# The 'apt-get install' of nvinfer-runtime-trt-repo-*
+# adds a new list which contains libnvinfer library, so it needs another
+# 'apt-get update' to retrieve that list before it can actually install the
+# library.
+# We don't install libnvinfer-dev since we don't need to build against TensorRT,
+# and libnvinfer4 doesn't contain libnvinfer.a static library.
+RUN apt-get update && \
+    apt-get install --no-install-recommends \
+        nvinfer-runtime-trt-repo-ubuntu1604-${TF_TENSORRT_VERSION}-ga-cuda10.0 && \
+    apt-get update && \
+    apt-get install --no-install-recommends \
+        libnvinfer5=${TF_TENSORRT_VERSION}-1+cuda10.0 && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/* && \
+    rm /usr/lib/x86_64-linux-gnu/libnvinfer_plugin* && \
+    rm /usr/lib/x86_64-linux-gnu/libnvcaffe_parser* && \
+    rm /usr/lib/x86_64-linux-gnu/libnvparsers*
+
+# nginx + njs
+RUN \
+    apt-get update && \
```

Comment:
Can we combine three apt-get update in one ? 


see this https://forums.docker.com/t/dockerfile-run-apt-get-install-all-packages-at-once-or-one-by-one/17191

Reply:
The apt-get are logically grouped together. Also this creates separate layers that can be cached separately so it speeds up 'docker build' during development.